clang/lib/CodeGen/CGCall.cpp

```diff
@@ -4310,10 +4309,12 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
 
     if (getLangOpts().CheckedC)
       IRFuncTy = cast<llvm::FunctionType>(TypeFromVal);
-    else
+    else {
+#ifndef NDEBUG
```

Comment:
If this is going to be a consistent merge conflict for future upgrades, do we want to document that here?